src/main/java/io/stargate/sgv2/jsonapi/exception/mappers/ConstraintViolationExceptionMapper.java

```diff
@@ -76,12 +77,24 @@ private static String valueDescription(Object rawValue) {
       return "<JSON value of " + rawValue.toString().length() + " characters>";
     }
 
+    if (rawValue instanceof FindCommand) {
+      if (propertyPath.contains("options.skip")) {
+        return String.format("%s", ((FindCommand) rawValue).options().skip());
+      }
+      if (propertyPath.contains("options.limit")) {
+        return String.format("%s", ((FindCommand) rawValue).options().limit());
+      }
+      if (propertyPath.contains("options.pageState")) {
+        return String.format("%s", ((FindCommand) rawValue).options().pageState());
+      }
+    }
+
```

Comment:
There is nowhere for me to know why the validation fails, so I pass the `propertyPath` and check it here. @tatu-at-datastax Any better idea?

Reply:
You could just include all values in a description, without trying to check which are relevant?

Reply:
So, we could have a message like this to include all the values in `options`? 
```
{
    "errors": [
        {
            "message": "Request invalid: field 'command.options.skip' value 'Options[limit=null, skip=10, pageState=null, includeSimilarity=false]' not valid. Problem: skip option should be used with sort clause.",
            "errorCode": "COMMAND_FIELD_INVALID"
        }
    ]
}
```